### PR TITLE
Added README to packages

### PR DESCRIPTION
**Pull request by cschuchardt88**
This gets rid of the warning when packing nuget packages, and helps users on nuget as well. So they know what the package is and does.

**Change Log**
- Added `README.md`
- Updated `Directory.Build.props`

**Removes**
```
The package Neo.3.6.2-CI01441 is missing a readme. Go to https://aka.ms/nuget/authoring-best-practices/readme to learn why package readmes are important.
```

### CODE REVIEW

**Comment by cschuchardt88** [comment]
@superboyiii @shargon This is a simple fix, please have a look.

File: src/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+## Overview
```
Comment (shargon) [inline src/README.md:1]: We can use the main Readme

**Review by shargon — COMMENTED** [review]
Currently nugget didn't work, so first we need to fix that

File: src/README.md (hunk 1 above)
Comment (cschuchardt88) [reply src/README.md:1]: It uses html in the file. I dont think you will be able to use it.

Comment (cschuchardt88) [reply src/README.md:1]: because has to work in visual studio and other readers

**Comment by shargon** [comment]
Blocked by https://github.com/neo-project/neo/issues/3024

**Comment by cschuchardt88** [comment]
>Blocked by #3024

Not merging isn't going to fix `myget`. So development and merging is stopped? We don't even need `myget`. Its mono-repo now, should be no reason to use it.

**Review by shargon — APPROVED** [review]

**Pull request merged**